Dockerfile

```diff
@@ -73,4 +73,4 @@ SHELL [ "/bin/bash", "-lc" ]
 
 # Set the default command to run when starting the container
 
-CMD conda activate hummingbot && ./bin/hummingbot_quickstart.py 2>./logs/standard_error_output.txt
\ No newline at end of file
+CMD conda activate hummingbot && ./bin/hummingbot_quickstart.py 2>> ./logs/errors.log
```

Comment:
@fengtality I reverted the changes because in the last commit you changed ./bin/hummingbot.py for ./bin/hummingbot_quickstart.py